Libraries/Blob/Blob.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule Blob
+ * @flow
+ */
+'use strict';
+
+const { BlobModule } = require('NativeModules');
+
+import type { BlobProps } from './BlobTypes';
+
+const UUID4 = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx';
+function createUUID(): string {
```

Comment:
Is there a tiny npm module you could use for this? Is the implementation copied from somewhere?